guides/v2.2/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -59,11 +62,120 @@ To enable Elasticsearch:
    bin/magento cache:clean
    ```
 
+## Elasticsearch software compatibility
+
+When you install or upgrade your {{ site.data.var.ece }} project, always check for compatibility between the Elasticsearch service version and the [Elasticsearch PHP](https://github.com/elastic/elasticsearch-php) client for {{ site.data.var.ee }}.
+
+-  **First time setup**–Confirm that the Elasticsearch version specified in the `services.yaml` file is compatible with the Elasticsearch PHP client configured for {{ site.data.var.ee }}.
+
+-  **Project upgrade**–Verify that the Elasticsearch PHP client in the new Magento version is compatible with the Elasticsearch service version installed on the Cloud infrastructure.
+
+{% include cloud/cloud-elasticsearch-client-compatibility.md %}
+
+{:.procedure}
+To check Elasticsearch software compatibility:
+
+1. Use SSH to log in to the remote environment.
+
+1. Check the Composer package version for `elasticsearch/elasticsearch`.
+
+   ```bash
+   composer show elasticsearch/elasticsearch
+   ```
+
+   In the response, check the installed version in the `versions` property.
+
+   ```terminal
+   name     : elasticsearch/elasticsearch
+   descrip. : PHP Client for Elasticsearch
+   keywords : client, elasticsearch, search
+   versions : * v6.7.1
+   type     : library
+   license  : Apache License 2.0 (Apache-2.0) (OSI approved) https://spdx.org    licensesApache-2.0.html#licenseText
+   source   : [git] https://github.com/elastic    elasticsearch-php.git7be453dd36d1b141b779f2cb956715f8e04ac2f4
+   dist     : [zip] https://api.github.com/repos/elastic/elasticsearch-php/zipball/     7be453dd36d1b141b779f2cb956715f8e04ac2f4 7be453dd36d1b141b779f2cb956715f8e04ac2f4
+   path     : /app/vendor/elasticsearch/elasticsearch
+   names    : elasticsearch/elasticsearch
+   ```
+
+   Also, you can find the Elasticsearch PHP client version in the  `composer.lock` file in the environment root directory.
+
+1. From the command line, retrieve the Elasticsearch service connection details.
+
+   ```bash
+   vendor/bin/ece-tools env:config:show services
+   ```
+
+   In the response, find the IP address for the Elasticsearch service endpoint:
+
+   ```terminal
+   | elasticsearch:                                                                                                  |
+   +------------------------------------------+----------------------------------------------------------------------+
+   | username                                 | null                                                                 |
```

Comment:
I would manually remove some of the white space from the first column. It really takes up a lot of screen space and might be annoying for someone on a mobile device.